src/Resources/skeleton/resetPassword/ResetPasswordController.tpl.php

```diff
@@ -155,9 +155,6 @@ private function processSendingPasswordResetEmail(string $emailFormData, MailerI
             '<?= $email_field ?>' => $emailFormData,
         ]);
 
-        // Marks that you are allowed to see the app_check_email page.
-        $this->setCanCheckEmailInSession();
```

Comment:
One small change that this introduces: Now an unknown email won't get to the check-email page since the next `if` clause will exit the function before `setTokenObjectInSession` can be called (can't be called anyway without a token).

This means that you'll be redirected in the `checkEmail` function (without user feedback) since no token is present. Is this an intentional change? Right now you'd know if the email existed by checking if a redirect occurs or not 🙂